src/lightning/pytorch/loggers/neptune.py

```diff
@@ -39,8 +39,8 @@
 log = logging.getLogger(__name__)
 
 # neptune is available with two names on PyPI : `neptune` and `neptune-client`
-_NEPTUNE_AVAILABLE = RequirementCache("neptune>=1.0")
-_NEPTUNE_CLIENT_AVAILABLE = RequirementCache("neptune-client")
+_NEPTUNE_AVAILABLE = bool(RequirementCache("neptune>=1.0")) or bool(RequirementCache("neptune-client>=1.0"))
+_LEGACY_NEPTUNE_CLIENT = bool(RequirementCache("neptune-client<1.0"))
```

Comment:
```suggestion
_LEGACY_NEPTUNE_CLIENT_AVAILABLE = bool(RequirementCache("neptune-client<1.0"))
```
to follow the naming pattern